.github/workflows/test-broken-content-latest.yml

```diff
@@ -15,6 +15,7 @@ jobs:
           [
             "new_kubeletconfig",
             "unexistent_resource",
+            "runtime_sshd",
```

Comment:
We should be able to remove this once https://github.com/ComplianceAsCode/content/pull/14118 lands?